include/zephyr/drivers/sensor/veml7700.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2022 Andreas Kilian
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_VEML7700_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SENSOR_VEML7700_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Number of enumerators in enum veml7700_als_gain.
+ */
+#define VEML7700_ALS_GAIN_IDX_MAX 4
+/**
+ * @brief Number of enumerators in enum veml7700_als_it.
+ */
+#define VEML7700_ALS_IT_IDX_MAX	6
```

Comment:
These macros should be renamed because they don't actually represent the max index, which is one less than the number of enumerators.